erica_app/erica/request_processing/eric_mapper.py

```diff
@@ -0,0 +1,197 @@
+    person_a_last_name: str
+    person_a_first_name: str
+    person_a_religion: str
+    person_a_street: str
+    person_a_street_number: str
+    person_a_street_number_ext: Optional[str]
+    person_a_address_ext: Optional[str]
+    person_a_plz: str
+    person_a_town: str
+
+    person_a_pauschbetrag_disability_degree: Optional[int]
+    person_a_pauschbetrag_has_merkzeichen_bl_tbl_h_pflegegrad: Optional[bool]
+    person_a_pauschbetrag_has_merkzeichen_g_ag: Optional[bool]
+    person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad: Optional[bool]
+    person_a_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80: Optional[bool]
+
+    person_b_same_address: Optional[bool]
+    person_b_idnr: Optional[str]
+    person_b_dob: Optional[str]
+    person_b_last_name: Optional[str]
+    person_b_first_name: Optional[str]
+    person_b_religion: Optional[str]
+    person_b_street: Optional[str]
+    person_b_street_number: Optional[str]
+    person_b_street_number_ext: Optional[str]
+    person_b_address_ext: Optional[str]
+    person_b_plz: Optional[str]
+    person_b_town: Optional[str]
+
+    person_b_pauschbetrag_disability_degree: Optional[int]
+    person_b_pauschbetrag_has_merkzeichen_bl_tbl_h_pflegegrad: Optional[bool]
+    person_b_pauschbetrag_has_merkzeichen_g_ag: Optional[bool]
+    person_b_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad: Optional[bool]
+    person_b_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80: Optional[bool]
+
+    stmind_haushaltsnahe_entries: Optional[List[str]]
+    stmind_haushaltsnahe_summe: Optional[Decimal]
+    stmind_handwerker_entries: Optional[List[str]]
+    stmind_handwerker_summe: Optional[Decimal]
+    stmind_handwerker_lohn_etc_summe: Optional[Decimal]
+
+    stmind_vorsorge_summe: Optional[Decimal]
+    stmind_spenden_inland: Optional[Decimal]
+    stmind_spenden_inland_parteien: Optional[Decimal]
+    stmind_religion_paid_summe: Optional[Decimal]
+    stmind_religion_reimbursed_summe: Optional[Decimal]
+
+    stmind_krankheitskosten_summe: Optional[Decimal]
+    stmind_krankheitskosten_anspruch: Optional[Decimal]
+    stmind_pflegekosten_summe: Optional[Decimal]
+    stmind_pflegekosten_anspruch: Optional[Decimal]
+    stmind_beh_aufw_summe: Optional[Decimal]
+    stmind_beh_aufw_anspruch: Optional[Decimal]
+    stmind_bestattung_summe: Optional[Decimal]
+    stmind_bestattung_anspruch: Optional[Decimal]
+    stmind_aussergbela_sonst_summe: Optional[Decimal]
+    stmind_aussergbela_sonst_anspruch: Optional[Decimal]
+
+    stmind_gem_haushalt_count: Optional[int]
+    stmind_gem_haushalt_entries: Optional[List[str]]
+
+    @root_validator(pre=True)
+    def set_pauschbetrag_disability_degrees(cls, values):
+        values['person_a_pauschbetrag_disability_degree'] = values.get('person_a_disability_degree')
+        values['person_b_pauschbetrag_disability_degree'] = values.get('person_b_disability_degree')
+
+        return values
```

Comment:
I think we should only do this of person_X_requests_pauschbetrag is True?

Reply:
Actually, same for all the other root validators. But maybe I'm confused and this is done somewhere else?